app/assets/javascripts/validation.coffee

```diff
@@ -349,6 +347,15 @@ $(document).ready ->
               params: {}
             errors.push newError
 
+  validateLatLonRanges = ->
```

Comment:
do you think this can be done in the `visitField` function on line 536?

Reply:
Now moved to visitField